cmd/kubelet/app/server.go

```diff
@@ -268,6 +272,8 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.ReallyCrashForTesting, "really-crash-for-testing", s.ReallyCrashForTesting, "If true, when panics occur crash. Intended for testing.")
 	fs.Float64Var(&s.ChaosChance, "chaos-chance", s.ChaosChance, "If > 0.0, introduce random client errors and latency. Intended for testing. [default=0.0]")
 	fs.BoolVar(&s.Containerized, "containerized", s.Containerized, "Experimental support for running kubelet in a container.  Intended for testing. [default=false]")
+	fs.BoolVar(&s.ReconcileCIDR, "reconcile-cidr", s.ReconcileCIDR, "Reconcile node CIDR with the CIDR specified by the API server. No-op if register-node or configure-cbr0 is false. [default=true]")
```

Comment:
Here is the reason: We removed salt-based of concileCIDR completely on all nodes, no matter master node or worker nodes, then master node's cidr bridge won't be configured by Kubelet at all. Introducing this one is to reconfige cidr for minion per apiserver. 

I think we can eventually consolidate some of those flags, but ok as it is today. :-)